samples/word/05-tables/table-cell-access.yaml

```diff
@@ -1,6 +1,7 @@
+order: 1
 id: word-tables-table-cell-access
-name: Table cell access
-description: Shows how to access a specific cell in a table.
+name: Creating and accessing a table
```

Comment:
Consider changing "Creating" to "Create" in snippet name (for consistency with how other snippet names start out)